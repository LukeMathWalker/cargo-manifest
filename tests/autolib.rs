use cargo_manifest::Manifest;

#[test]
fn autolib() {
    let m = Manifest::from_path("tests/autolib/Cargo.toml").expect("load autolib");
    insta::assert_debug_snapshot!(m);
}
