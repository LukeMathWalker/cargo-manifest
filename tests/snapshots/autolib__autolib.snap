---
source: tests/autolib.rs
expression: m
---
Manifest {
    package: Some(
        Package {
            name: "auto-lib",
            edition: None,
            version: Some(
                Local(
                    "0.1.0",
                ),
            ),
            build: None,
            workspace: None,
            authors: None,
            links: None,
            description: None,
            homepage: None,
            documentation: None,
            readme: None,
            keywords: None,
            categories: None,
            license: None,
            license_file: None,
            repository: None,
            metadata: None,
            rust_version: None,
            exclude: None,
            include: None,
            default_run: None,
            autobins: true,
            autoexamples: true,
            autotests: true,
            autobenches: true,
            publish: None,
            resolver: None,
        },
    ),
    cargo_features: None,
    workspace: None,
    dependencies: None,
    dev_dependencies: None,
    build_dependencies: None,
    target: None,
    features: None,
    bin: [],
    bench: [],
    test: [],
    example: [],
    patch: None,
    lib: Some(
        Product {
            path: Some(
                "src/lib.rs",
            ),
            name: Some(
                "auto_lib",
            ),
            test: true,
            doctest: true,
            bench: true,
            doc: true,
            plugin: false,
            proc_macro: false,
            harness: true,
            edition: None,
            required_features: [],
            crate_type: Some(
                [
                    "lib",
                ],
            ),
        },
    ),
    profile: None,
    badges: None,
}
